scripts/xtensa-build-zephyr.py

```diff
@@ -349,9 +357,16 @@ def build_platforms():
 		if args.cmake_args:
 			build_cmd += args.cmake_args
 
+		overlays = []
+		if "CONFIG_OVERLAY" in platform_dict:
+			overlays.append(str(pathlib.Path(SOF_TOP, "platforms", platform, platform_dict["CONFIG_OVERLAY"])))
```

Comment:
I don't think it's possible to have an entirely empty Kconfig file, at the very least we need `CONFIG_ICELAKE=y` or equivalent.

So can you make `overlay.conf` the default filename and avoid the repetition? Something like this:
```python
overlay_filename = platform_dict.get("CONFIG_OVERLAY", default="overlay.conf")
overlays.append(str(pathlib.Path(SOF_TOP, "platforms", platform, overlay_filename))
```

Reply:
Yes sure. I was wondering if it is possible to build without Kconfig entries at all just by what is given in board files but if it is not lets remove it.